src/test/webhook.spec.ts

```diff
@@ -1,18 +1,48 @@
-import { chai, expect, app, StatusCodes } from "./config/server.config";
-import { deleteAllWebhooks } from "./mocks/webhook.mocks";
+import {
+  chai,
+  expect,
+  sinon,
+  app,
+  StatusCodes,
+  RecipeDocument,
+  User,
+} from "./config/server.config";
+import { deleteAllWebhooks, addWebhook } from "./mocks/webhook.mocks";
+import { getToken, deleteAllUsers } from "./mocks/user.mocks";
+import {
+  addSomeRecipes,
+  deleteAllRecipes,
+  recipes,
+} from "./mocks/recipe.mocks";
+import webhookService from "../services/webhook.service";
+import { assert, SinonMatcher } from "sinon";
 
-afterEach("Delete all users", async function () {
-  await deleteAllWebhooks();
-});
+describe("Webhook testing", function () {
+  let token: string;
+  let userId: string;
+  let webhookId: string;
+
+  beforeEach("Add webhook, user and get token", async function () {
+    const user = await getToken();
```

Comment:
`getToken()` returns `user`?

Reply:
@arkus7 `getToken()` adds user to DB, signs token and returns object containing token and userId. I can't find any other name for it - createUserAndReturnTokenAndUserId or getTokenAndUserId? 😆